src/features/resume/FinalView.js

```diff
@@ -1,72 +1,88 @@
 import { useSelector } from 'react-redux';
 import React from 'react';
+import TemplateOne from '../../components/TemplateOne';
 import '../../App.css';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { library } from "@fortawesome/fontawesome-svg-core";
+import { library } from '@fortawesome/fontawesome-svg-core';
 import { faPhone, faGraduationCap, faBriefcase } from '@fortawesome/free-solid-svg-icons';
 
-library.add( faPhone, faGraduationCap, faBriefcase);
+library.add(faPhone, faGraduationCap, faBriefcase);
 
 const FinalView = () => {
 	const resumeData = useSelector((state) => state.resume.resumeData);
-	const pickedTemplate = useSelector((state) => state.resume.style);
-	const { firstName, lastName, email, phone, education, bio, linkedin, location, socialWeb, title } = resumeData;
+	const pickedTemplate = useSelector((state) => state.resume.template);
+	const {
+		firstName,
+		lastName,
+		email,
+		phone,
+		education,
+		bio,
+		linkedin,
+		location,
+		socialWeb,
+		title,
+	} = resumeData;
 
 	return (
-	<div className="wrapper">
-		<div className={pickedTemplate}>
-			<h5 className={`${pickedTemplate}__name`}>
-				{firstName} {lastName} CV
-			</h5>
-			<h5 className={`${pickedTemplate}__title`}>
-				{title}
-			</h5>
-			<div className={`${pickedTemplate}__about`}>
-				<p className={`${pickedTemplate}__bio`}>{bio}</p>
-			</div>
-			<div className={`${pickedTemplate}__section`}>
-				<div className={`${pickedTemplate}__basic`}>
-					<h1><FontAwesomeIcon icon={"fa-regular", "phone"} /></h1>
-					<div>
-						<h1 className="header">Contact</h1>
-							<p className="header__text">
-								{phone}
-							</p>
-						<p className="header__text">
-							{location}
-						</p>
-						<p className="header__text">
-							{email}
-						</p>
-						<p className="header__text">
-							{socialWeb}
-						</p>
-						<p className="header__text">
-							{linkedin}
-						</p>
+		<div className='wrapper'>
+			{pickedTemplate === `first-template` ? (
+				<TemplateOne pickedTemplate={pickedTemplate} resumeData={resumeData} />
+			) : (
+				<div className={pickedTemplate}>
+					<h5 className={`${pickedTemplate}__name`}>
```

Comment:
in this case pickedTemplate doesn't make sense if it the other template is abstracted from the main view

Reply:
Is this the template as a component? Your template should also be made into a component shouldn't it?

Reply:
Okay, I'll merge then.